src/Mutation/MediaItemCreate.php

```diff
@@ -102,8 +102,9 @@ public static function get_input_fields() {
 	public static function get_output_fields() {
 		return [
 			'mediaItem' => [
-				'type'    => 'MediaItem',
-				'resolve' => function ( $payload, $args, AppContext $context ) {
+				'type'        => 'MediaItem',
+				'description' => 'The MediaItem object mutation type.',
```

Comment:
We need to wrap this in translation function

```suggestion
				'description' => __( 'The MediaItem object mutation type.', 'wp-graphql' ),
```